spec/controllers/playlists_controller_spec.rb

```diff
@@ -560,4 +560,20 @@
         end
       end
     end
+
+    describe "GET #manifest" do
+      let(:playlist) { FactoryBot.create(:playlist, items: [playlist_item], visibility: Playlist::PUBLIC) }
+      let(:playlist_item) { FactoryBot.create(:playlist_item, clip: clip) }
+      let(:clip) { FactoryBot.create(:avalon_clip, master_file: master_file) }
+      let(:master_file) { FactoryBot.create(:master_file, media_object: media_object) }
+      let(:media_object) { FactoryBot.create(:published_media_object, visibility: 'public') }
+
+      it "returns a IIIF manifest" do
+        get :manifest, format: 'json', params: { id: playlist.id }, session: valid_session
+        expect(response).to have_http_status(200)
+        parsed_response = JSON.parse(response.body)
+        expect(parsed_response['@context']).to include "http://iiif.io/api/presentation/3/context.json"
+        expect(parsed_response['type']).to eq 'Manifest'
+      end
```

Comment:
I think it would be good to include some expectations about the content of the manifest.  Maybe it would be sufficient to check that `items` is not empty?